.github/workflows/build_and_publish.yaml

```diff
@@ -72,6 +72,8 @@ jobs:
     - name: Generate correct value for VERSION file
       run: |
         echo ${{ needs.tag-new-version.outputs.tag }} > cohortextractor/VERSION
+    - name: Pull image
+      run: docker pull docker-proxy.opensafely.org/opensafely/cohortextractor
```

Comment:
This should be ghcr.io/ rather than the proxy, as its running it gh. We should only need to use the proxy to pull stuff onto the server.